grin/src/adapters.rs

```diff
@@ -85,14 +91,15 @@ impl NetAdapter for NetToChainAdapter {
 					      bh.hash(),
 					      bh.height,
 					      s);
+					return Err(p2p::Error::Invalid);					
```

Comment:
This gets returned when a peer sent a valid header but at the wrong time (too early/late with respect to our current chain state). We should not penalize the peer for that.